src/components/RuntimeManager.js

```diff
@@ -108,6 +110,50 @@ const RuntimeErrorNotification = ({ runtime }) => {
   ])
 }
 
+export const AppErrorModal = ({ app, onDismiss }) => {
+  const [error, setError] = useState()
+  const [loadingAppDetails, setLoadingAppDetails] = useState(false)
+
+  const loadAppError = _.flow(
+    withErrorReporting('Error loading app details'),
+    Utils.withBusyState(setLoadingAppDetails)
+  )(async () => {
+    const { errors: appErrors } = await Ajax().Apps.app(app.googleProject, app.appName).details()
+    setError(appErrors[0].errorMessage)
```

Comment:
should this be `appErrors[0]?.errorMessage` to guard against when there are none (one of those 'shouldn't happen but why not be safe' things)